### DISCUSSION TITLE
Simulating the night Sky over Titanic immediately before the collision

### DISCUSSION THREAD

**Discussion by Pchapel** [Stellarium/stellarium#3083] [General]
Hi everyone

I need some guidance in entering the Milky Way brightness and saturation, and the Zodiacal light brightness on the night of this famous collision with an iceberg. For the Milky Way I'm using 5/10 for brightness and 0.5 out of 1 for saturation.  For the Zodiacal Light I'm using 2 out of 10.  However, aren't these factors automatically varied depending upon the area of the Milky way and time of night?  I assumed the Zodiacal light was only present just before twilight in the early morning or after twilight in the early evening in the direction of the Sun, but Stellarium seems to put some in in by default, despite it being locally around midnight

Obviously, we have to go off eyewitness accounts which can be varied, but most describe a night sky of exceptional clarity where stars can be seen right down to the horizon (little extinction). Others describe a mist, all around.  Some have speculated this mist was a 'Fata Morgana' mirage.  However, could this be airglow, and would it peak much lower than the customary 10 degrees on a night like this, and how does Stellarium allow for airglow?  Also does Stellarium mix all the elements (stars, planets, milky way, airglow) to create natural skyglow?  Obviously I've set natural light pollution to zero, since it was customary on ships of the time not to compromise eye night sensitivity using forward facing searchlights, or any light forward of the bridge or crows nest which are kept in total darkness.

It's relatively straightforward to enter the time and location  41 46.2 N 49 55.0 W which is the best estimate of the collision coordinates, whilst the time is set to 15 April 1912 at 02:37:23 GMT (Julian year 2419507.6092940). Note these are not the Wreck coordinates or ships time which are different.  The ship is headed at 266 degrees true, straight at the star Procyon.  Below it, to the horizon was the dim constellation of Monoceros which is crossed by the Milky Way.

Using these figures I've compiled an image of the Iceberg silhouette against the Milky way and Zodiacal light at the average distance the iceberg was first sighted.  Note, from the height of the crows nest, the Iceberg doesn't obstruct the sky directly, only it's reflection in the sea which was described as exceptionally flat, and without waves. One book describes it as a sea of glass.  Testimony from the Lookouts says there was no white tip or reflection from the iceberg until near, only a black mass against the sea, although other experts suggested this was unlikely. For the purposes of this discussion I'll start with that assumption, then add what they might have actually seen later.

![Silhouette of reflection of Iceberg](https://user-images.githubusercontent.com/127088914/223069483-ae0a0b0d-e350-4623-8403-03c74410afff.jpg)

**Comment by 10110111** [comment]
> However, aren't these factors automatically varied depending upon the area of the Milky way and time of night?

These factors are applied on top of those that vary with natural conditions.

> I assumed the Zodiacal light was only present in the twilight in the direction of the Sun, but Stellarium seems to put it in by default.

Zodiacal light is brighter near the direction of the Sun, but strictly speaking, the light scattered by the dust (which results in Zodiacal light and Gegenschein) is not only there. It's present over the whole ecliptic, as you can see [in this photo](https://en.wikipedia.org/wiki/Zodiacal_light#/media/File:False_Dawn.jpg):

![](https://upload.wikimedia.org/wikipedia/commons/5/5a/False_Dawn.jpg)

> However, could this be airglow, and would it peak much lower than the customary 10 degrees on a night like this, and how does Stellarium allow for airglow? Also does Stellarium mix all the elements (stars, planets, milky way, airglow) to create natural skyglow?

Stellarium doesn't simulate airglow other than by some ad hoc constants in the logic that decides on the visibility of stars. And no, I don't think airglow would be brighter near the horizon, because it originates high in the atmosphere and then undergoes extinction just like starlight. Just see [this photo](https://en.wikipedia.org/wiki/File:Panoramic_shot_of_the_VLT_platform.jpg):

![](https://upload.wikimedia.org/wikipedia/commons/c/c8/Panoramic_shot_of_the_VLT_platform.jpg)

Generally, from my experience, what I see in Stellarium doesn't usually match the perception of brightnesses in the real sky. The stars are more or less OK, but airglow, color of the night sky, etc. are not quite realistic. Moreover, AFAICT, the very dim objects are not even rendered in physically-based luminance units, instead all this is done with ad hoc constants that "looked good enough" for those who developed it.

**Comment by gzotti** [comment]
Stars, Milky way and Zodiacal light (ZL) should mix OK with their weights set at 1.0. However, users are free to exaggerate parts of the night sky if they want to concentrate on particular features. There is also always the problem of physical screen brightness and environmental conditions of the monitor/screen. 
ZL is not part of twilight! It's interplanetary dust.
Stellarium has no separate simulation of airglow. Night sky brightness follows Schaefer's model, at least then the old skylight model is in use.
Landscapes can have a "fog" layer to visually simulate low mist, but rather decoratively. This predates proper atmospheric extinction computation. 

After many years of simulating sky colors with a common daylight model, one of us has introduced a much more ambitious skylight model, which may however need some more rebalancing to let the night sky appear more natural. This is an art that needs time and observing practice under dark sky conditions.

**Reply by Pchapel** [reply]
It's important not to exaggerate as I'm trying to determine what the lookouts would exactly see.  Most reports suggest the stars were exceptionally bright right down to the horizon.  The Key Constellation here is Monoceros, which isn't along the Ecliptic, but the fainter part of the Galactic plane. The contrast of a dark object with the background is key here.

I don't have any reflection of icebergs against the Milky Way, but here's a photo of a tree reflected in a lake against the Milky Way, but **adjusted down to resemble what the eye can actually see**.  So not those long exposure photos, of which there are plenty!

![milky way with silhouette of tree](https://user-images.githubusercontent.com/127088914/223088285-5ec3a4a5-74c6-45fb-a68d-5593d3bf269e.jpg)

**Reply by gzotti** [reply]
Use a large, not too bright monitor in a dark room or even out under the real sky dome in a dark-sky area. Switch light pollution off. Observe for 30 minutes to let your eyes adjust. See if setting the various mixing values to 1 does justice. See whether setting absorption coefficient to 0.15 does justice to absorption (per textbook models it should not be less at sea level!).  If not, adjust. I don't know the Titanic conditions.

**Comment by Pchapel** [comment]
This link includes a barometric chart. It was cold, becalmed under high pressure.
https://www.accuweather.com/en/weather-blogs/weathermatrix/weather-maps-from-the-night-the-titanic-sank/1173542

a more technical paper on the meteorological conditions is here.
https://www.researchgate.net/publication/277680147_Reconstructing_the_Prevailing_Meteorological_and_Optical_Environment_during_the_Time_of_the_Titanic_Disaster

**Comment by Pchapel** [comment]
I've assumed this statement about Airglow from Wikipedia

"Although airglow emission is fairly uniform across the atmosphere, it appears brightest at about 10° above the observer's horizon, since the lower one looks, the greater the mass of atmosphere one is looking through. Very low down, however, atmospheric extinction reduces the apparent brightness of the airglow"
https://en.wikipedia.org/wiki/Airglow#Description

However this statement from a passenger suggests to me that extinction was very low on the night.

"The complete absence of haze produced a phenomenon I had never seen before: where the sky met the sea the line was as clear and definite as the edge of a knife, so that the water and the air never merged gradually into each other and blended to a softened rounded horizon, but each element was so exclusively separate that where a star came low down in the sky near the clear-cut edge of the water-line, it still lost none if its brilliance. As the earth revolved and the water edge came up and covered partially the star, as it were, it simply cut the star in two"

[https://timmaltin.com/2015/06/10/lawrence-beesley-thermal-inversion/]

Incidentally, Aurora was reported by some passengers. At 41N I think that very unlikely, but could it have been airglow they were seeing? Another possibility is that they just saw a flare from the Carpathia, rescue ship.

**Reply by 10110111** [reply]
If there indeed was temperature inversion as described in your link, the horizon could indeed have a repeated/stretched image of the upper sky instead of the extincted "correct", i.e. unmiraged, horizon. This could then be just the usual airglow, without the need to invoke an aurora.

**Comment by Atque** [comment]
How accurate are eye-witness accounts by non-astronomers in a life-threatening situation?

**Reply by Pchapel** [reply]
Probably not very, especially since several accounts are contradictory. 

The Lookouts and Officers would have been very familiar with the night sky though, the latter using it directly for navigation.  I doubt if there are many opportunities nowadays to see such dark skies.

**Comment by Pchapel** [comment]
> _"The complete absence of haze produced a phenomenon I had never seen before: where the sky met the sea the line was as clear and definite as the edge of a knife, so that the water and the air never merged gradually into each other and blended to a softened rounded horizon, but each element was so exclusively separate that where a star came low down in the sky near the clear-cut edge of the water-line, it still lost none if its brilliance. As the earth revolved and the water edge came up and covered partially the star, as it were, it simply cut the star in two"_

What conditions would be necessary for no appreciable extinction to occur?  Surely, either the horizon would need to be raised relative to the stars, or the stars would need to be depressed relative to the horizon.

A bit like the vanishing line in this explanation.  the only problem is that this is an inferior mirage which is caused by a layer of cold air above the warm.  There was a warm current to the South, but I think this was some tens of miles away
http://hyperphysics.phy-astr.gsu.edu/hbase/atmos/mirage.html

![image](https://user-images.githubusercontent.com/127088914/224366854-87249935-fa2c-4ab3-97b7-98005e57d52c.png)
